allen-cell-animated/timelapse-colorizer
# fix: Cookie consent banner styling flashes

Problem
=======
Fixes an issue where the styling overrides for the cookie banner would not appear on initial load, and would instead flash through the default style.

*Estimated review size: tiny, <5 minutes*

Solution
========
- Imports `cookie-banner.css` in the `index.html` for the app.
- Removes nesting syntax for backwards compatibility
- Fixes some buttons and headers that were not being styled.

## Type of change
* Bug fix (non-breaking change which fixes an issue)

Steps to Verify:
----------------
1. Open the main build. The styling will flash briefly. https://timelapse.allencell.org
2. Open the preview link. The style override should not flash. https://allen-cell-animated.github.io/timelapse-colorizer/pr-preview/pr-430/
